build.js

```diff
@@ -26,7 +26,6 @@ let entries = {
   "certified-search-results": "./static/js/src/certified-search-results.js",
   openstackDeploymentChart: "./static/js/src/openstack-deployment-chart.js",
   blender: "./static/js/src/advantage/subscribe/blender/app.tsx",
-  tabbedContent: "./static/js/src/tabbed-content.js",
```

Comment:
Why is this being removed? Many pages have this dependency?

Reply:
It is needed in the navigation, so I made it globally available via main.js. If you check some other pages they should still be working fine.